build-configs/aschaffenburg/index.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+import { lightTheme, darkTheme } from './theme'
+import type {
+  AndroidBuildConfigType,
+  CommonBuildConfigType,
+  iOSBuildConfigType,
+  WebBuildConfigType
+} from '../BuildConfigType'
+import { ASCHAFFENBURG_ASSETS } from '../AssetsType'
+import mainImprint from './mainImprint'
+
+const APPLE_APP_ID = ''
+
+const commonAschaffenburgBuildConfig: CommonBuildConfigType = {
+  appName: 'Aschaffenburg App',
+  appIcon: 'app_icon_integreat',
```

Comment:
I think we should TODOs in places where not the right data is not available yet.